python_ta/checkers/simplifiable_if_checker.py

```diff
@@ -0,0 +1,42 @@
+"""Checker that checks for nested if statements that can be simplified"""
+
+from __future__ import annotations
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages
+from pylint.lint import PyLinter
+
+
+class SimplifiableIfChecker(BaseChecker):
+    """A checker class that reports nested if statements that can be simplified"""
+
+    name = "simplifiable-if"
+    msgs = {
+        "E9930": (
+            "This nested `if` statement can be simplified. Add this test condition to the branch above and remove the nested `if` statement.",
```

Comment:
"Add this test condition to the branch above" -> "Combine the inner condition with the outer condition using the `and` operator and remove the nested `if` statement"